recipes/_configure_server.rb

```diff
@@ -132,6 +132,25 @@
   }
 end if license_uri == 'self'
 
+bag_license = CernerSplunk::DataBag.load(node['splunk']['config']['license-pool'])
+
+if node['splunk']['node_type'] == :license_server && !bag_license.nil?
+  auto_generated_pool_size = bag_license['auto_generated_pool_sizeMB']
+  server_stanzas['lmpool:auto_generated_pool_enterprise']['quota'] = auto_generated_pool_size.to_s + 'MB'
```

Comment:
@acharlieh recommended writing the size in raw bytes (the way splunk do it, if we do it from the UI) rather than writing it in MB. I will make the change soon.